tubular/scripts/frontend_utils.py

```diff
@@ -207,6 +216,52 @@ def deploy_site(self, bucket_name, app_path):
         return_code = proc.wait()
         if return_code != 0:
             self.FAIL(1, 'Could not sync app {} with S3 bucket {}.'.format(self.app_name, bucket_uri))
+
+    def _upload_js_sourcemaps(self, app_path):
+        """ Upload JavaScript sourcemaps to Datadog. """
+        app_config = self.get_app_config()
+        api_key = app_config.get('DATADOG_API_KEY')
```

Comment:
[inform/context] 

```
```
# Can't upload source maps without ``DATADOG_API_KEY``, which must be set as an environment variable
# before executing the Datadog CLI. The Datadog documentation suggests using a dedicated Datadog API key:
# https://docs.datadoghq.com/real_user_monitoring/guide/upload-javascript-source-maps/
```
```

It looks like `DATADOG_API_KEY` exists in edx-internal as an ansible secret (e.g., [here](https://github.com/edx/edx-internal/blob/48eb9028aa9c4ed7bc4846ba692470dff7fb35da/ansible/vars/edx.yml#L17)), but I'm not too sure whether this is accessible by the frontend configs used in `frontend_utils.py`.

As is, I'm attempting to extract `DATADOG_API_KEY` from the MFE `APP_CONFIG`. Assumes `DATADOG_API_KEY` is defined under the common frontend config (e.g., [here](https://github.com/edx/edx-internal/blob/48eb9028aa9c4ed7bc4846ba692470dff7fb35da/frontends/common/prod_config.yml#L1), shared across all MFEs.